.changeset/plenty-forks-act.md

```diff
@@ -0,0 +1,6 @@
+---
+'@sap-ux-private/preview-middleware-client': minor
+'@sap-ux/preview-middleware': minor
```

Comment:
Please change from `minor` to `patch`. This is a small change, we do not need a minor version update

Reply:
Done